contrib/babelfishpg_tsql/src/pl_exec-2.c

```diff
@@ -2955,6 +2959,225 @@ exec_stmt_insert_bulk(PLtsql_execstate *estate, PLtsql_stmt_insert_bulk *stmt)
 	return PLTSQL_RC_OK;
 }
 
+int exec_stmt_dbcc(PLtsql_execstate *estate, PLtsql_stmt_dbcc *stmt)
+{
+	switch (stmt->dbcc_stmt_type)
```

Comment:
If we don't expect the function to be called for other dbcc statement type, we should add an assert(0) for the default case.
```
case default:
		Assert(0);
  
```

Reply:
Made changes in the latest commits, Thanks.